File: implementation/scipamato/core/entity/src/test/java/ch/difty/scipamato/core/entity/code_class/CodeClassTranslationTest.java
package ch.difty.scipamato.core.entity.code_class;

import static org.assertj.core.api.Assertions.assertThat;

import org.junit.jupiter.api.Test;

class CodeClassTranslationTest {

    @Test
    void comment() {
        CodeClassTranslation cct = new CodeClassTranslation(1, "de", "code1", "description", 1);
        assertThat(cct.getDescription()).isEqualTo("description");
    }

    @Test
    void displayValue() {
        CodeClassTranslation cct = new CodeClassTranslation(1, "de", "code1", "description", 1);
        assertThat(cct.getDisplayValue()).isEqualTo("de: code1");
    }

    @Test
    void field() {
        assertThat(CodeClassTranslation.CodeClassTranslationFields.DESCRIPTION.getFieldName()).isEqualTo("description");
    }
}

File: implementation/scipamato/core/entity/src/test/kotlin/ch/difty/scipamato/core/entity/code_class/CodeClassTranslationTest.kt
package ch.difty.scipamato.core.entity.code_class

import org.assertj.core.api.Assertions.assertThat

import org.junit.jupiter.api.Test

internal class CodeClassTranslationTest {

    @Test
    fun comment() {
        val cct = CodeClassTranslation(1, "de", "code1", "description", 1)
        assertThat(cct.description).isEqualTo("description")
    }

    @Test
    fun displayValue() {
        val cct = CodeClassTranslation(1, "de", "code1", "description", 1)
        assertThat(cct.displayValue).isEqualTo("de: code1")
    }

    @Test
    fun field() {
        assertThat(CodeClassTranslation.CodeClassTranslationFields.DESCRIPTION.fieldName).isEqualTo("description")
    }
}